.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
intentional to check in?

Reply:
No, but we should reconsider our gitignore if we don't want this to happen. settings.json is explicitly allowed.

Reply:
Deleted for now.